project-api/project-api-test/src/test/resources/myproject_with_lang_import/main.bal

```diff
@@ -0,0 +1,7 @@
+import myproject.services;
```

Comment:
Add license header

Reply:
please find the header below 
```
// Copyright (c) 2024 WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
//
// WSO2 Inc. licenses this file to you under the Apache License,
// Version 2.0 (the "License"); you may not use this file except
// in compliance with the License.
// You may obtain a copy of the License at
//
// http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing,
// software distributed under the License is distributed on an
// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
// KIND, either express or implied.  See the License for the
// specific language governing permissions and limitations
// under the License.
```

Reply:
> please find the header below
> 
> ```
> // Copyright (c) 2024 WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
> //
> // WSO2 Inc. licenses this file to you under the Apache License,
> // Version 2.0 (the "License"); you may not use this file except
> // in compliance with the License.
> // You may obtain a copy of the License at
> //
> // http://www.apache.org/licenses/LICENSE-2.0
> //
> // Unless required by applicable law or agreed to in writing,
> // software distributed under the License is distributed on an
> // "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
> // KIND, either express or implied.  See the License for the
> // specific language governing permissions and limitations
> // under the License.
> ```

should i use it as it is or change the website URL domain to .com?

Reply:
You need to change it to `.com`

Reply:
added the license headers in 95dc03b438c8c66c9f0ccf3fa841e548ff1688a9 and 03f85ee8b7376df74bc8cf76b9dc7681d0900647 commits.